packages/liveblocks-core/src/connection.ts

```diff
@@ -207,10 +207,6 @@ function enableTracing(machine: FSM<Context, Event, State>) {
     machine.events.willTransition.subscribe(({ from, to }) => {
       log("Transitioning", from, "→", to);
     }),
-    machine.events.didPatchContext.subscribe((patch) => {
-      log(`Patched: ${JSON.stringify(patch)}`);
-      // log(`New context: ${JSON.stringify(machine.context)}`);
-    }),
```

Comment:
This event isn't really needed, it mostly adds noise, because the only moments the context _can_ actually be patched is during transitions or onEnter/onExit handlers, which we already fire events for.